frontend/components/organisms/OChangePasswordForm.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Paper, Grid, Divider, Button } from '@material-ui/core';
+import { useState, ChangeEvent } from 'react';
+import MPasswordField from '../molecules/MPasswordField';
+
+const OChangePasswordForm = () => {
+  const [currPass, setCurrPass] = useState('');
+  const [newPass, setNewPass] = useState('');
+  const [confirmPass, setConfirmPass] = useState('');
```

Comment:
Add validation when they submit to check if these two are the same, if not warn the user. You can refer to `OLoginDialog`.